package.json

```diff
@@ -13,5 +13,14 @@
     "body-parser": "^1.19.0",
     "express": "^4.17.1",
     "mongoose": "^5.9.28"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^9.1.2",
+    "husky": "^4.3.0"
```

Comment:
Im not sure that `typicode/husky` team fix this issue, but after version `3.0.8` it stopped to work properly with sourcetree. I think we should downgrade this version to `3.0.8`

Reply:
Could you check this? I use a git via original cli and I have no problems with it.

Reply:
Via CLI it works fine, but in external GUI it can be skipped. To check